### ISSUE TITLE
CollectRnaSeqMetrics report very low rRNA RIBOSOMAL_BASES

### ISSUE THREAD

**Issue by wangyugui** [broadinstitute/picard#841]
### Affected tool(s)
CollectRnaSeqMetrics 

### Affected version(s)
- 2.8.3 /2.9.4 (tested)

### Description 
CollectRnaSeqMetrics report very low rRNA RIBOSOMAL_BASES value.
but we can confirm there are a lot of reads(NOT 0x900) in rRNA region.

#### Steps to reproduce

CollectRnaSeqMetrics command detail is reported in RnaSeqMc.txt.
[RnaSeqMc.txt](https://github.com/broadinstitute/picard/files/1082828/RnaSeqMc.txt)

```
#bedtools multicov -excl-flags 0x900 -bams ./Aligned.sortedByCoord.out.bam -bed rRNA.bed
...
1       228639334       228639455       RNA5S14 .       -       0
1       228641565       228641686       RNA5S15 .       -       0
1       228643806       228643927       RNA5S16 .       -       2
1       228646037       228646158       RNA5S17 .       -       0
MT      647     1601    RNR1    .       +       245896
MT      1670    3229    RNR2    .       +       891431
NT_167214.1     105423  118780  RNA45S5 .       +       14911050
```

**Comment by meganshand** [comment]
@wangyugui Could you please share the rRNA interval list you are using?

**Comment by wangyugui** [comment]
[rRNA.intervals.txt](https://github.com/broadinstitute/picard/files/1085407/rRNA.intervals.txt)

We use RNA 45S as one unit, so RRNA_FRAGMENT_PERCENTAGE is set to 0.001.

**Comment by sooheelee** [comment]
Hi @wangyugui, 

Going forward, please post Picard questions to the GATK forum <http://gatkforums.broadinstitute.org/gatk/discussions>. 

I will try to help you as best I can. 0x900 is not a SAM flag I am aware of. Is it new? The last on my list is hexadecimal 0x800 (2048 decimal) for SUPPLEMENTARY alignments. See [this blogpost](http://gatkforums.broadinstitute.org/gatk/discussion/7019/sam-flags-down-a-boat) and [SAM specs](https://samtools.github.io/hts-specs/SAMv1.pdf).

**Comment by wangyugui** [comment]
Hi.

1) 0x200 flag
I checked 0x200 flag,  and no reads is reported by STAR 2.5.3 in this region with 0x200 flags.

```
# bedtools multicov -incl-flags 0x200 -bed ${gff%/*}/rRNA.bed -bams ./Aligned.sortedByCoord.out.bam
...
MT      647     1601    RNR1    .       +       0
MT      1670    3229    RNR2    .       +       0
NT_167214.1     105423  118780  RNA45S5 .       +       0
```

2)NT_167214.1
NT_167214.1 is included in RefSeq GRCh38.p10 (GCF_000001405.36) . and RNA45S5 is also included in GFF file of GCF_000001405.36.

3)REFFLAT 
I'm using RefSeq GRCh38.p10, so I convert gff to refFlat
                gff3ToGenePred -useName ${gff} /dev/stdout | genePredExt2refFlat.awk >refFlat.txt

```
# tail /usr/bio-ref/GRCh38.p10/refFlat.txt
XR_001737584.1  XR_001737584.1  1       -       91168   120798  120798  120798  3       91168,92090,120774,     91629,92240,120798,
XR_001737583.1  XR_001737583.1  1       -       91168   168093  168093  168093  4       91168,92090,165883,167963,      91629,92240,165942,168093,
```

**Comment by sooheelee** [comment]
Sorry @wangyugui, I'm used to following up in the forum. Did you solve your issue? If not, can you submit a test case for us?

**Comment by wangyugui** [comment]
@sooheelee 

I have not solved this issue. but I get rRNA from 
1)the ReadsPerGene file of STAR's output.
2)rsem output

There are some human samples in NCBI that have diff rRNA rate.
10%-20% Level:SAMN03465419,SAMN03465420,SAMN03465421
over 50% :SAMN03568911,SAMN03568912
less than 1%:  SAMN04284274

**Comment by wangyugui** [comment]
It seems be fixed in picard 2.10.3-SNAPSHOT.

There are no other result about other picard version

This is a  test result of 100bp PE RNA lib in my local.

Picard Result:
```
PF_BASES        PF_ALIGNED_BASES        RIBOSOMAL_BASES CODING_BASES
12478171423     11015462445             2808418470
```


ReadsPerGene file of STAR's output
```
RNA45S5 851261  660537  190728
RNA5S1  0       0       0
RNA5S2  0       0       0
...
RNA5S17 0       0       0
RNR1    575318  90365   484993
RNR2    13281800        2074726 11211629
pairs_rRNA/pairs_total(%)       14708379        56773212        25.9073
```

**Comment by sooheelee** [comment]
Glad to see Picard v2.10.3 fixes your issue.
